src/igb/patch/0002-force-disable-mii-isolate-mode-for-bcm54616.patch

```diff
@@ -0,0 +1,62 @@
+Force disable MII isolate mode
+
+From: wadelnn <wadelnn@users.noreply.github.com>
+---
+ src/e1000_82575.c |  3 +++
+ src/e1000_phy.c   | 13 +++++++++++++
+ src/e1000_phy.h   |  1 +
+ 3 files changed, 17 insertions(+)
+
+diff --git a/src/e1000_82575.c b/src/e1000_82575.c
+index 049ab7b..97e7025 100644
+--- a/src/e1000_82575.c
++++ b/src/e1000_82575.c
+@@ -1592,6 +1592,9 @@ static s32 e1000_setup_copper_link_82575(struct e1000_hw *hw)
+ 		case I210_I_PHY_ID:
+ 			ret_val = e1000_copper_link_setup_m88_gen2(hw);
+ 			break;
++		case BCM54616_E_PHY_ID:
++			e1000_disable_mii_isolate_bcm54616(hw);
++			break;
+ 		default:
+ 			ret_val = e1000_copper_link_setup_m88(hw);
```

Comment:
I am not convinced that this patch does not affect other nic that also use 54616 phy since in previous code, they will call e1000_copper_link_setup_m88. Now, they will only call your function.